src/main/java/com/uber/cadence/internal/worker/ActivityPollTask.java

```diff
@@ -91,6 +92,10 @@ public ActivityPollTask(
     }
 
     options.getMetricsScope().counter(MetricsType.ACTIVITY_POLL_SUCCEED_COUNTER).inc(1);
+    options
+        .getMetricsScope()
+        .timer(MetricsType.ACTIVITY_SCHEDULED_TO_START_LATENCY)
+        .record(Duration.ofNanos(System.nanoTime() - result.scheduledTimestamp));
```

Comment:
result.getStartedTimestamp - result.getScheduledTimestampOfThisAttempt

Reply:
addressed. also fixed go client in https://github.com/uber-go/cadence-client/pull/836